src/main/scala/uk/gov/hmrc/releaser/Releaser.scala

```diff
@@ -208,10 +209,12 @@ class Releaser(stageDir:Path,
                connectorBuilder:(RepoFlavour) => RepoConnector,
                coordinator:Coordinator){
 
-  def start(artefactName: String, gitRepo:Repo, rcVersion: ReleaseCandidateVersion, targetVersionString: ReleaseVersion): Try[Unit] = {
+  def start(artefactName: String, artefactSuffix : String, gitRepo:Repo, rcVersion: ReleaseCandidateVersion, targetVersionString: ReleaseVersion): Try[Unit] = {
 
-    repositoryFinder(artefactName) flatMap { repo =>
-      val ver = VersionMapping(repo, artefactName, gitRepo, rcVersion, targetVersionString)
+    val artName = s"$artefactName$artefactSuffix"
```

Comment:
@duncancrawford Silly question... can you not just pass in the artefact name including the suffix to achieve the same result?